helper-cli/src/main/kotlin/HelperMain.kt

```diff
@@ -72,6 +73,7 @@ internal class HelperMain : CliktCommand(name = ORTH_NAME, epilog = "* denotes r
         }
 
         subcommands(
+            ConvertToDependencyGraphCommand(),
```

Comment:
Is there still the plan to eventually also convert in the other direction, from graph to tree, e.g. to make result better comparable / diffable? I believe the outcome from our discussion was "no, we want to give diffing the graph format a try first", but I'm still wondering whether we should generalize the name of the command, like "ConvertDependencies" or "ConvertDependencyRepresentation", and then from the option(s) tell from which direction into which. Or simply auto-detect it, and always convert into the other format than what is in the input.

Reply:
Maybe consider also "SetDependencyRepresentationCommand".

Reply:
I liked Frank's suggestion to renamed the command to `SetDependencyRepresentationCommand`.

Regarding the output format: I prefer explicitly specifying the target format of the conversion. If the command always did a conversion, you would lose the use case to ensure that a file has a specific format by just running the command on it. Therefore, I added an option to set the target format based on an enum. This is currently a bit moot, as there is only a single supported output format, but can be extended easily.

Reply:
> If the command always did a conversion, you would lose the use case to ensure that a file has a specific format by just running the command on it.

Good point.